components/fonts/platform/freetype/font_list.rs

```diff
@@ -209,17 +221,15 @@ pub(crate) fn default_system_generic_font_family(
     generic: GenericFontFamily,
 ) -> LowercaseFontFamilyName {
     let generic_string = match generic {
-        GenericFontFamily::None | GenericFontFamily::Serif => "serif",
-        GenericFontFamily::SansSerif => "sans-serif",
-        GenericFontFamily::Monospace => "monospace",
-        GenericFontFamily::Cursive => "cursive",
-        GenericFontFamily::Fantasy => "fantasy",
-        GenericFontFamily::SystemUi => "sans-serif",
+        GenericFontFamily::None | GenericFontFamily::Serif => b"serif\0".as_slice(),
+        GenericFontFamily::SansSerif => b"sans-serif\0".as_slice(),
+        GenericFontFamily::Monospace => b"monospace\0".as_slice(),
+        GenericFontFamily::Cursive => b"cursive\0".as_slice(),
+        GenericFontFamily::Fantasy => b"fantasy\0".as_slice(),
+        GenericFontFamily::SystemUi => b"sans-serif\0".as_slice(),
     };
```

Comment:
You can use the c-str literal syntax `c"a zero terminated cstr"`

Reply:
I forgot that was a thing, thanks!